sctp.c

```diff
@@ -133,8 +126,8 @@ static void janus_sctp_association_free(const janus_refcount *sctp_ref) {
 	sctp = NULL;
 }
 
-janus_sctp_association *janus_sctp_association_create(void *dtls, uint64_t handle_id, uint16_t udp_port) {
-	if(dtls == NULL || udp_port == 0)
+janus_sctp_association *janus_sctp_association_create(void *dtls, struct janus_ice_handle* ice, uint16_t udp_port) {
```

Comment:
There's no need for the `struct` part, and please invert the whitespace and the asterisk (code style).
As a side note, since we're explicitly referring to the handle type, it's probably worth to stop referring to `dtls` as a pointer to void and use the proper type as well. This would allow us to also add a reference to `dtls` as well here as we do for the handle: we had a reference before, but with the thread gone it's now missing, meaning we should add it here.

Reply:
Done (`dtls` already was refcounted by casting it).